src/main/java/Runner.java

```diff
@@ -25,17 +27,21 @@
             new MilitaryPlane("C-130 Hercules", 650, 5000, 110000, MilitaryType.TRANSPORT)
     );
 
-    public static void main(String[] args) throws NonMilitaryPlainException {
-        Airport airport = new Airport(planes);
-        Airport militaryAirport = new Airport(airport.getMilitaryPlanes());
-        Airport passengerAirport = new Airport(airport.getPassengersPlane());
-        System.out.println("Military airport sorted by max distance: " + militaryAirport
-                .sortByMaxDistance()
-                .toString());
-        System.out.println("Passenger airport sorted by max speed: " + passengerAirport
-                .sortByMaxSpeed()
-                .toString());
-
-        System.out.println("Plane with max passenger capacity: " + passengerAirport.getPassengerPlaneWithMaxPassengersCapacity());
+    public static void main(String[] args) {
```

Comment:
main method should be only an entry point to the program. it's good that you've caught exception but please separate.
main should only call logic. if not creating ad additional class (no need for such small learning app), at least create objects in main, then perform a logic on them in separate method which will handle expections.